README.md

```diff
@@ -8,20 +8,20 @@
 > A simple wrapper around [winston](https://github.com/winstonjs/winston).
 
 ### Installation
-NPM: `npm install enrise-logger --save`  
+NPM: `npm install enrise-logger --save`
 Yarn: `yarn add enrise-logger`
 
 ### Initialization and usage
-At the beginning of your application, be sure to initialize the logger:  
+At the beginning of your application, be sure to initialize the logger:
 `require('enrise-logger')([config: Object]);`
 
 Where `config` is an optional object. See [below](#configuration) for further instructions.
 
-After the module is initialized, simply call `.get(name: String)` on the module to return a namespaced logger:  
```

Comment:
The double spaces create a return in the rendered Readme, which IMO improved readability for all of these cases. (Except the one in the list, it didn't make sense there)